src/main/kotlin/org/opensearch/replication/task/index/IndexReplicationTask.kt

```diff
@@ -828,7 +828,11 @@ open class IndexReplicationTask(id: Long, type: String, action: String, descript
             }
         } catch(e: Exception) {
             val err = "Unable to initiate restore call for $followerIndexName from $leaderAlias:${leaderIndex.name}"
+            val aliasErrMsg = "cannot rename index [${leaderIndex.name}] into [$followerIndexName] because of conflict with an alias with the same name"
```

Comment:
Relying on exception message isn't robust as any simple change in string will impact the logic.

Have we exhausted other ways to detect? (Exception type, error code etc). If so, we may want to check on substring instead of complete sentence.

Reply:
This is just to show specific error message in case of alias conflict. Main concern is to handle failed state in case of restore state. If string will change in future as well, then also we get the failed state with message related to "unable to initiate restore" and more specific error will come in logs